automation_tools/__init__.py

```diff
@@ -1255,19 +1255,6 @@ def install_prerequisites():
     manage_daemon('start', 'ntpd', warn_only=True)
 
 
-def install_puppet4(puppet4_repo=None):
-    """Task to install puppet4 prior Satellite installation"""
-    # Remove any puppet3 server to avoid conflicts
-    run('yum -y remove puppet-server', warn_only=True)
-    create_custom_repos(puppet4_repo=puppet4_repo)
```

Comment:
this one remained so I removed whole task and put this one in place of install_puppet4()